index.js

```diff
@@ -51,6 +51,7 @@ module.exports = function (di, directory) {
             helper.requireWrapper('pluralize', 'pluralize'),
             helper.requireWrapper('always-tail', 'Tail'),
             helper.requireWrapper('flat', 'flat'),
+            helper.requireWrapper('ajv', 'ajv'),
```

Comment:
Change the injectable Name to `Ajv` will give clear message for user that it is a Class not instance.